app/src/main/java/ch/epfl/skysync/navigation/HomeGraph.kt

```diff
@@ -4,17 +4,54 @@ import androidx.navigation.NavGraphBuilder
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.composable
 import androidx.navigation.navigation
+import ch.epfl.skysync.models.UNSET_ID
+import ch.epfl.skysync.models.calendar.TimeSlot
+import ch.epfl.skysync.models.flight.FlightType
+import ch.epfl.skysync.models.flight.PlannedFlight
+import ch.epfl.skysync.screens.AddFlightScreen
 import ch.epfl.skysync.screens.CalendarScreen
 import ch.epfl.skysync.screens.ChatScreen
 import ch.epfl.skysync.screens.FlightScreen
 import ch.epfl.skysync.screens.HomeScreen
+import java.time.LocalDate
 
 /** Graph of the main screens of the app */
 fun NavGraphBuilder.homeGraph(navController: NavHostController) {
+  val listFlights =
```

Comment:
You should add a comment precising that this is for preview purposes only so that you can get back to it easily when integrating the backend :)